pyproject.toml

```diff
@@ -148,6 +147,10 @@ editorconfig-checker = "2.4.0"
 types-requests = "^2.28"
 boto3-stubs = "^1.26"
 
+# This is no longer required for normal parsec operation
+# but tests may use it
```

Comment:
Why can't we also remove oscrypto use from the tests ?

Reply:
I am not sure, it was previously mentioned in an exchange between you and @TimeEngineer so I assumed it was true, but never checked by myself.

I'll be on it.

Reply:
We can, but need more effort

Reply:
It is used here:

```python
def test_only_rsa_is_supported():
    # Unsupported format for service encryption key (only RSA is currently supported)
    unsupported_public_key, _ = oscrypto.asymmetric.generate_pair("dsa", bit_size=1024)
    with pytest.raises(ValueError):
        SequesterPublicKeyDer(
            oscrypto.asymmetric.dump_public_key(unsupported_public_key, encoding="der")
        ),
    with pytest.raises(ValueError):
        SequesterVerifyKeyDer(
            oscrypto.asymmetric.dump_public_key(unsupported_public_key, encoding="der")
        ),
```

To check we do not handle DSA keys. I lack context on this, is this still revelant today ?

If it is relevant, I can modify the test to remove this dependency on oscrypto.

If however it is not the case, we may drop the test. What would be the best course of action ?

Reply:
I will migrate this

Reply:
Done